src/Components/Endpoints/test/Builder/RazorComponentsEndpointConventionBuilderExtensionsTest.cs

```diff
@@ -209,13 +209,38 @@ public void WithStaticAssets_DoesNotAddResourceCollectionFromGroup_WhenMappingNo
 
         // Assert
         var groupEndpoints = Assert.IsAssignableFrom<IEndpointRouteBuilder>(group).DataSources;
-        Assert.All(groupEndpoints.Skip(2).First().Endpoints, e =>
+        Assert.All(groupEndpoints.Skip(1).First().Endpoints, e =>
         {
             var metadata = e.Metadata.GetMetadata<ResourceAssetCollection>();
             Assert.Null(metadata);
         });
     }
 
+    [Fact]
+    public void MapRazorComponents_CanAddConventions_ToBlazorWebEndpoints()
```

Comment:
Do we have a similar test for `RedirectionEndpointBaseRelativeUrl`? Or can we verify we can add metadata to that too in the same test?

Reply:
I've added a test co cover redirection endpoint base relative URL too.